kani-compiler/src/kani_middle/transform/loop_contracts.rs

```diff
@@ -32,6 +32,42 @@ pub struct LoopContractPass {
     new_loop_latches: HashMap<usize, usize>,
 }
 
+//Useful for debugging
+#[allow(dead_code)]
+pub fn print_stable_mir_body(body: &Body) {
+    println!("=== MIR Body Start ===");
+    // Print basic blocks
+    for (idx, block) in body.blocks.iter().enumerate() {
+        println!("bb{idx}: {{");
+        // Print statements
+        for stmt in &block.statements {
+            println!(" {stmt:?}");
+        }
+        // Print terminator
+        println!(" {:?}", block.terminator);
+        println!("}}");
+    }
+    println!("=== MIR Body End ===");
+}
+
+//Useful for debugging
+#[allow(dead_code)]
+pub fn print_mutablebody(body: &MutableBody) {
+    println!("=== MIR Body Start ===");
+    // Print basic blocks
+    for (idx, block) in body.blocks().iter().enumerate() {
+        println!("bb{idx}: {{");
+        // Print statements
+        for stmt in &block.statements {
+            println!(" {stmt:?}");
+        }
+        // Print terminator
+        println!(" {:?}", block.terminator);
+        println!("}}");
+    }
+    println!("=== MIR Body End ===");
+}
+
```

Comment:
I acknowledge those might be useful, but can we please have them in some place that is a better fit for general utilities independent of loop contracts?